docs/source/guides/Data-sharing.md

```diff
@@ -0,0 +1,76 @@
+
+## Data
+### Repository
+There are [many repositories](repo-table) that will accept the types of data generated by SWC. They come into three categories, general, neuroscience-specific and modality-specific.  If you do not have a preference, we recommended:
+
+**For small (<20GB) datasets:**
+Use the existing [SWC section of the UCL FigShare instance](https://rdr.ucl.ac.uk/The_Sainsbury_Wellcome_Centre). This is easy to use, and is supported centrally by UCL
+
+**For larger datasets:**
+Create a GIN repository, ideally within the [SWC organisation](https://gin.g-node.org/SainsburyWellcomeCentre). GIN uses Git and provides a versioning system, making it particularly well suited for ongoing research projects.
+
+**Specific modalities:**
+In some cases, there may be a modality or field-specific repository that would be best suited for the data (or mandated by the publisher), some examples include:
+- [Brain Image Library](http://www.brainimagelibrary.org/) - Large brain image datasets
+- [DANDI](https://dandiarchive.org/) - Electrophysiology, optophysiology, behavioural time-series (must be in NWB format)
+- [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/) - Biological images
+
+In all cases, using a repository that can create a [DOI](https://www.doi.org/) will help increase citations.
+
+### Structure
+There is no single best way to structure your data, but it must be in a form that allows users to easily re-use it for their own purposes. This may be by adopting specific file formats (such as [NeuroData Without Borders](https://www.nwb.org/)). Alternatively the data could be organised within a standardised project folder structure. Many labs have their own system, but those starting from scratch may wish to use the [SWC-Blueprint data structure](https://swc-blueprint.neuroinformatics.dev/).
+
+**However the data is structured, all relevant metadata must be included with the raw data.**
+
+## Assistance
+If you have any technical questions about sharing data or code, please contact adam.tyson@ucl.ac.uk.
+
+(repo-table)=
+## Reference of potential repositories for neuroscience data
+| Repository                         |URL                                                |Domain                                                                                               |
+|------------------------------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------|
+| Globus                             |                                                   |Anything                                                                                             |
```

Comment:
Missing link for globus?

Reply:
There isn't a link for Globus really, it's a way of sharing data we have internally. Useful for collaborations, but not really for public sharing. It's here for completeness really.